watcher_dispatcher/README.md

```diff
@@ -0,0 +1,13 @@
+# Watcher - Dispatcher
+
+## Description
+
+This is the watcher - dispatcher service for the proving system.
+
+The architecture consists of a watcher thats monitors the blockchain for new transactions and a dispatcher that sends these new transactions to the provers. It could be pluged to any blockchain that supports smart contracts.
```

Comment:
small nit, I think it would we more correct if instead of "new transactions" we say "confirmed blocks".
Little typo: pluged -> plugged ✅

Reply:
fixed, thanks!